pandas/tests/indexes/conftest.py

```diff
@@ -1,8 +1,30 @@
+import numpy as np
 import pytest
 
 import pandas._testing as tm
 from pandas.core.indexes.api import Index, MultiIndex
 
+
+def _gen_mi():
+    # a MultiIndex used to test the general functionality of the
```

Comment:
I think "general functionality" is mistakenly repeated twice

Reply:
thanks, will update